### PR TITLE
Add GET method for TriggerCollection

### PR DESCRIPTION
**Pull request by lxwinspur**
PR to grab support for Triggers collection: 

Note: The changes here were pulled from gerrit upstream, and just solved some conflicts.

Upstream changes:
Add GET method for TriggerCollection: https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/45490

### CODE REVIEW

**Comment by gtmills** [comment]
jenkins run tests please

File: scripts/update_schemas.py [hunk 1]
```diff
@@ -11,6 +11,102 @@
 
 VERSION = "DSP8010_2021.2"
 
+# To use a new schema, add to list and rerun tool
```
Comment (gtmills) [inline scripts/update_schemas.py:14]: We removed this downstream. Don't need to do anything with this file

Comment (lxwinspur) [reply scripts/update_schemas.py]: Done

Comment (lxwinspur) [reply scripts/update_schemas.py]: > Please fix CI here as well

Done

**Comment by gtmills** [comment]
Going to close. We might reopen at a later time if we do need triggers.

**Pull request closed without merging**